project/Dependencies.scala

```diff
@@ -77,6 +86,11 @@ object Dependencies {
     lazy val partyManagement =
       namespace %% "interop-selfcare-party-management-client" % partyManagementClientVersion
 
+    lazy val partyProcessClient = namespace %% "interop-selfcare-party-process-client" % partyProcessVersion
```

Comment:
we are going to migrate our services to the newer version, if possible we should directly use v2 of selfcare APIs